src/assets/cart/ExternalLink.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There should be already an external link image

Reply:
I'd like to have it for file centric cart only.